crates/vm/levm/src/vm.rs

```diff
@@ -138,6 +138,7 @@ impl VM {
 
         let mut default_touched_storage_slots: HashMap<Address, HashSet<H256>> = HashMap::new();
 
+        // Add access lists contents to cache
```

Comment:
Is this meant for this PR?

Reply:
It's not strictly related, but makes more understandable the code IMO